website/docs/tutorials/08-cairo-1-support/01-requirements.md

```diff
@@ -6,37 +6,22 @@ sidebar_label: Requirements
 
 ## Project structure
 
-:::note
-Cairo 1 is constantly being updated, so all the requirements mentioned here may change in the future.
-:::
+Projects should follow a defined structure. Learn more in the [project initialization](./02-project-initialization.md)
+section.
 
-Currently, Cairo1 forces the following file structure of a contract:
-```
-my_contract/
-├── lib.cairo
-├── cairo_project.toml
-```
+## Module and crate names
 
-That means that you can have only one contract per module. More information about modules [here](./02-project-initialization.md#cairo-1-modules).
-
-- `my_contract` - a root directory for a contract
-    - `cairo_project.toml` which is needed for compilation. Its purpose is to specify a path to the crate roots of the contract.
-    This file usually looks like this:
-    ```
-    [crate_roots]
-    my_contract = "."
-    ```
-    - `lib.cairo` file which defines the module and may also contain the code of the contract.
-
-## Module name
-
-The name must use only ASCII alphanumeric characters or `_`, and cannot be empty. It cannot also start with underscore. For the alternative reference, please see [this](https://docs.swmansion.com/scarb/docs/reference/manifest#name).
+The names must use only ASCII alphanumeric characters or `_`, and cannot be empty. It cannot also start with underscore.
+For the alternative reference, please see [this](https://docs.swmansion.com/scarb/docs/reference/manifest#name).
```

Comment:
This section does not make much sense without introducing modules before

Reply:
How about introducing some sort of "glossary" at the beginning of this file? We could then keep detailed modules explanation in the `02-project-initialization`.

Reply:
On the second though, this could probably be merged with project initialization. Do you think that this is a better approach?

Reply:
@cptartur glossary is a bad idea it should be story-like explaining concepts on demand, my only issue here is we should introduce consistent names for those concepts (namely crates modules)

Reply:
I'll reorder this a bit and add some links.